src/Microsoft.Azure.WebJobs.Host/Tables/TableAttributeBindingProvider.cs

```diff
@@ -49,6 +52,143 @@ public TableAttributeBindingProvider(INameResolver nameResolver, IStorageAccount
                 new PocoEntityArgumentBindingProvider()); // Supports all types; must come after other providers
         }
 
+        // [Queue] has some pre-existing behavior where the storage account can be specified outside of the [Queue] attribute. 
```

Comment:
good catch

---
In reply to: [85970367](https://github.com/Azure/azure-webjobs-sdk/pull/891#discussion_r85970367) [](ancestors = 85970367)